src/rgw/rgw_object_expirer_core.h

```diff
@@ -4,12 +4,6 @@
 #ifndef CEPH_OBJEXP_H
 #define CEPH_OBJEXP_H
 
-#include <errno.h>
```

Comment:
one more nit, it seems that we're following the convention including std headers before self-defined headers.